frontend/src/store/app/selectors.ts

```diff
@@ -40,6 +40,10 @@ export const getBackendConnected = createSelector(
   getApp,
   (app: AppState): boolean => app.backendConnection,
 );
+export const getLastBackendConnectionTime = createSelector(
```

Comment:
I haven't gone through the entire PR, but I'm thinking maybe we can use this in the `useEffect` in `OverlayScreen` or if we replace that with the saga in the future to limit the dispatch of `BACKEND_CONNECTION_UP` action?

Reply:
That's an interesting idea! `lastBackendConnectionTime` changes exactly when a `BACKEND_CONNECTION_UP` action changes `backendConnection` from false to true, so whether it limits dispatch of the `BACKEND_CONNECTION_UP` action will depend on the structure of our conditionals in that `useEffect`.